.gitignore

```diff
@@ -11,3 +11,4 @@
 # Other
 *pycache*
 
+.vscode
```

Comment:
We have already updated `.gitignore` with possible options for IDE stuff so that we have no need in this update.